util/tls/tls_socket.cc

```diff
@@ -290,6 +317,8 @@ io::Result<size_t> TlsSocket::SendBuffer(Engine::Buffer buf) {
     }
 
     int op_val = *op_result;
+    LOG_IF(WARNING, spin_count.Check(op_val == 0))
```

Comment:
This can lead potentially to logs getting full right?

Reply:
No, look above what `Check` is doing internally. It resets the counter, so it acts as `LOG_EVERY_N`

Reply:
In this case N is 20 so if we have a client that is not reading from the socket for some reason we will write every 20 yields

Reply:
yes that's true. I wrote in a previous comment this is a heuristic, we can always adjust :shrug:

Reply:
I would either use log every T here or would change the spin count to grow exponentialy i,e first time print after x times next after x^2 times etc

Reply:
we can also use LOG_EVERY_N_SEC() with sec=600 for example..

Reply:
we dont use absl logging - LOG_EVERY_N_SEC
glog have LOG_EVERY_T which is the same